arbitrum-docs/node-running/running-a-node.mdx

```diff
@@ -104,19 +104,71 @@ Note: If you’re interested in accessing an Arbitrum chain, but you don’t wan
+  * `MakeNodes` (Whitelist required)
+    - Continuously create new nodes, challenging any bad assertions found (wallet required, always staked, most expensive node to run)
+    - Note that if there is more than one `MakeNodes` validator running, they might all try to create a new node at same time. In that case, only one will be successful, and the others will have still spent gas on reverted call that didn't do anything.
+
+### Running Watchtower Validator
+
+- Running a validator in `Watchtower` mode is the same as running an Nitro node with `--node.validator.enable --node.validator.strategy=Watchtower`
+- Here is an example of how to run a watchtower validator for Arbitrum One:
   ```shell
-  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum offchainlabs/nitro-node:v2.0.10-73224e3 --l1.url=https://l1-mainnet-node:8545 --l2.chain-id=42161 --node.caching.archive --node.validator.enable --node.validator.strategy=Watchtower
+  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum offchainlabs/nitro-node:v2.0.11-8e786ec --l1.url=https://l1-mainnet-node:8545 --l2.chain-id=42161 --node.validator.enable --node.validator.strategy=Watchtower
   ```
 - If a deviation is detected, a validator in Watchtower mode will log an error containing the string `found incorrect assertion in watchtower mode`
+- To verify validator is working, this log line shows the wallet is setup correctly:
+  ```shell
+  INFO [09-28|18:43:49.367] running as validator                     txSender=nil actingAsWallet=nil whitelisted=false strategy=Watchtower
+  ```
+  - `strategy` should be `Watchtower`
+  - The log line `validation succeeded` shows that the L2 block validator is working
+  - The log line `found correct assertion` shows that the L1 validator is working
+
+### Creating Wallet for Whitelisted Validator
+
+- Watchtower validators never need a wallet, because they never post on-chain
+- Defensive validators need a wallet configured, but the wallet does not need to be funded until it logs that an assertion has been found
+- All other validators require a funded wallet to immediately post stake, as well as additional funds that will be spent at regular intervals
+- Here is an example of how to tell Nitro to create validator wallet for Arbitrum One and exit:
+  ```shell
+  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum offchainlabs/nitro-node:v2.0.11-8e786ec --l1.url=https://l1-mainnet-node:8545 --l2.chain-id=42161 --node.validator.enable --l1.wallet.only-create-key --l1.wallet.password="SOME SECURE PASSWORD"
+  ```
+- Wallet file will be created under the mounted directory inside the `arb1/wallet/` directory for Arb1, or `nova/wallet/` directory for Nova.  Be sure to backup the wallet, it will be the only way to withdrawal stake when desired
```

Comment:
```suggestion
- Wallet file will be created under the mounted directory inside the `arb1/wallet/` directory for Arb1, or `nova/wallet/` directory for Nova.  Be sure to backup the wallet, it will be the only way to withdraw stake when desired
```